src/include/uapi/drm_local/amdxdna_accel.h

```diff
@@ -516,6 +516,20 @@ struct amdxdna_drm_aie_reg {
 	__u32 val;
 };
 
+/**
+ * struct amdxdna_drm_aie_coredump - The data for AIE coredump
+ * @context_id: The hw context id
+ * @size:  The size of bytes to read/write
+ * @buf_p: The buffer to store write data
+ *
+ * This is used for DRM_AMDXDNA_READ_AIE_COREDUMP parameters.
+ */
+struct amdxdna_drm_aie_coredump {
+	__u32 context_id;
+	__u32 size;
+	__u64 buf_p;
```

Comment:
@NishadSaraf reminds me that the nested buffer schema does not work on virtio environment. Can we change it to a flat buffer with a header (in our case, the header contains just context id).